uint32 num_sweeps
uint32 num_samples_per_sweep
uint32 num_elements
Complex[] radar_cube
 